### PR TITLE
Bootstrap tweaks, part 1 of 3

### PR DESCRIPTION
**Pull request by osheroff**
I've been working on how the bootstrap frontend operates.  This first PR is a minor refactor with a logging fix. 

@zendesk/rules @nmaquet

### CODE REVIEW

File: bin/maxwell-bootstrap [hunk 1]
```diff
@@ -4,7 +4,7 @@ base_dir=$(dirname $0)/..
 
 CLASSPATH=$CLASSPATH:$base_dir/target/classes:../open-replicator/target/classes
 
-for file in $base_dir/target/maxwell-*/*/lib/*.jar lib/*.jar
+for file in $base_dir/target/dependency/*.jar lib/*.jar
```
Comment (dadah89) [inline bin/maxwell-bootstrap]: don't you need to change `assembly.xml` to reflect this change?

**Comment by dadah89** [comment]
just a comment about bin/maxwell-bootstrap, everything else looks fine to me, :+1:

File: bin/maxwell-bootstrap (hunk 1 above)
Comment (osheroff) [inline bin/maxwell-bootstrap]: nope.  I'm done with Maven.  see the recent history of master, I've got a working `make` implementation of packaging.

Comment (dadah89) [inline bin/maxwell-bootstrap]: I see, still had an older version of Maxwell compiled locally

**Pull request merged**